curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62dabe2ef403a12d5d295273.md

```diff
@@ -46,7 +46,7 @@ The `p` element content visible in the browser should be `See more cat photos in
 ```js
 const pText = document
   .querySelector('p')
-  .innerText.toLowerCase()
+  .innerHTML.toLowerCase()
```

Comment:
```suggestion
  .innerText.toLowerCase()
```
You have to put this back to `innerText`. The `innerHTML` of the `p` element includes the anchor element inside it.